quickstart-query.ts

```diff
@@ -81,12 +94,22 @@ async function generativeSearchGroupedQuery() {
    { limit: 2 }
  );
 
+ for (let object of result.objects) {
+  console.log(JSON.stringify(object.properties, null, 2));
```

Comment:
(As above)

Reply:
not sure what you mean here

Reply:
Maybe I misread the change, I thought it was to display the distance field